schema/compound.yaml

```diff
@@ -46,6 +46,22 @@ links:
         type: float
 
 properties:
+  therapy_type:
+    type:
+      - 'null'
+      - string
+  regimen:
+    type:
+      - 'null'
+      - string
+  administration:
+    type:
+      - 'null'
+      - string
+  dose:
```

Comment:
I think dose is specific to an individual. The compound record points more to the concept of a chemical, rather then the application to an individual.